src/components/color-checkbox.tsx

```diff
@@ -2,9 +2,9 @@ import { ColorTypes } from "@/constants/color-type";
 import cn from "@/utils/style/cn";
 
 type props = {
-  color: ColorTypes[];
+  color: ColorTypes | ColorTypes[];
```

Comment:
```suggestion
  color: ColorTypes[];
```
như này thôi, phía dứoi check include, ko cần type

Reply:
Dạ tại có trường hợp là mảng color ( ở filter ) 
Có trường hợp lúc add to cart thì nó chỉ chọn 1 nên mới có trường hợp cả 2 cái ạ
Làm common cho cả 2 case nên em làm thế

Reply:
![image](https://github.com/user-attachments/assets/e623f3d4-f12d-424a-92f1-8d9010b0b374)